tests/fuzzer/fuzzer.py

```diff
@@ -458,7 +456,7 @@ def create_scenario(base_url: str, infra_id: int) -> Scenario:
         "name": "fuzzer_scenario",
         "infra": infra_id,
```

Comment:
```suggestion
        "infra_id": infra_id,
```

This function isn't run with pytest, it's only used when running the script by itself